src/structures/Message.js

```diff
@@ -52,13 +52,19 @@ class Message extends Base {
     if (data) this._patch(data);
   }
 
-  _patch(data, partial = false) {
+  _patch(data) {
     /**
      * The message's id
      * @type {Snowflake}
      */
     this.id = data.id;
 
+    /**
+     * The timestamp the message was sent at
+     * @type {number}
+     */
+    this.createdTimestamp = SnowflakeUtil.deconstruct(this.id).timestamp;
```

Comment:
Why doesn't this use data.timestamp instead?

Reply:
I suppose since the id is always there, that's one less thing that can break. (e.g. partial messages)